src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -449,11 +449,38 @@ public GHProject createProject(String name, String body) throws IOException {
 
     /**
      * The enum Permission.
+     *
+     * @deprecated Use <code>RolePermission</code> instead, which allows for custom permissions/roles created in github
+     *             to be set.
      */
+    @Deprecated
```

Comment:
It is very odd that this enum is only used for setting permissions not reading them.  Oh well, not your problem. :)